setup.py

```diff
@@ -51,7 +51,7 @@ def get_packages(package):
 
 
 extra_requirements = [
-    "websockets==8.*",
+    "websockets==9.*",
```

Comment:
```suggestion
    "websockets>=8.*",
```

Reply:
wouldn't it make sense to keep 8 as well, as it worked before?

Reply:
make sense!
Thank you!